include/deal.II/lac/linear_operator.h

```diff
@@ -821,6 +862,104 @@ namespace internal
         v.reinit(matrix.n(), omit_zeroing_entries);
       }
     };
+
+
+    /**
+    * A dummy class for LinearOperators that do not require any extensions
+    * to facilitate the operations of the matrix.
+    *
+    * This is the Payload class typically associated with deal.II's native
+    * SparseMatrix. To use Trilinos and PETSc SparseMatrices it is necessary
+    * to initialize a LinearOperator with their associated Payload.
+    *
+    * @author Jean-Paul Pelteret, 2016
+    *
+    * @ingroup LAOperators
+    */
+    class EmptyPayload
+    {
+    public:
+
+      /**
+      * Default constructor
+      */
+      EmptyPayload ()
+      {}
+
+      /**
+      * Specific constructor basen on an exemplary matrix
+      *
+      * It is from this constructor that we can safely
+      * build an operational instance of the payload
+      */
+      template<typename MatrixExemplar, typename Matrix>
+      EmptyPayload (const MatrixExemplar &matrix_exemplar,
+                    const Matrix         &matrix)
+      {
+        (void)matrix_exemplar;
+        (void)matrix;
+      }
+
+      /**
+      * Copy constructor
+      */
+      EmptyPayload (const EmptyPayload &rhs,
```

Comment:
`rhs` sounds wrong. This is simply a copy constructor that transposes the original object.